src/safari-app-extension/AlpheiosReadingTools/Base.lproj/MainMenu.xib

```diff
@@ -13,11 +13,14 @@
         </customObject>
         <customObject id="-1" userLabel="First Responder" customClass="FirstResponder"/>
         <customObject id="-3" userLabel="Application" customClass="NSObject"/>
-        <customObject id="Voe-Tx-rLC" customClass="AppDelegate" customModule="AlpheiosReadingTools" customModuleProvider="target">
+        <customObject id="Voe-Tx-rLC" customClass="AppDelegate" customModule="Alpheios_Reading_Tools" customModuleProvider="target">
```

Comment:
does the name change matter here?

Reply:
is it because the entitlements file name has changed?

Reply:
I don't remember when I was changing it specifically, probably when I was tweaking security permissions to make the Core Data access from both a containing app and an app extension work. This name is a result of the "Module" option set to "Inherit Module From Target":
![image](https://user-images.githubusercontent.com/18631055/64594249-b677d900-d3c0-11e9-949b-08c666fd8214.png)

Because the containing app target is "Alpheios Reading Tools" and probably spaces are not allowed and are replaced with underscores this results is the name as in the changed code.

I do not know much about the inner mechanics of this, and seems that very few people do. It seems almost not documented at all. But if you suspect that it might create issues for the App Store, please let me know, and I will investigate this.

Reply:
I have no clue what the app store will do :-) But I guess we will find out. Let's not worry about it for now.